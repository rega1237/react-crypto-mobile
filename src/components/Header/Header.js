import logo from '../../icon/logo.png';

const Header = () => (
  <header>
    <img src={logo} alt="company logo" />
  </header>
);

export default Header;
