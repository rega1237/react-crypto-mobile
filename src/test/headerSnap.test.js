import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from '../components/Header/Header';

test('Header is rendering', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Header />
      </MemoryRouter>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
