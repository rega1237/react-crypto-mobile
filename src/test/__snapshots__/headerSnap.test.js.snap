// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header is rendering 1`] = `
<header>
  <img
    alt="company logo"
    src="logo.png"
  />
</header>
`;
